sentinel-core/src/main/java/com/alibaba/csp/sentinel/node/Node.java

```diff
@@ -164,16 +164,18 @@
     /**
      * Increase the block count.
      *
+     * @param rt Current timestamp, calculated based on this
      * @param count count to add
      */
-    void increaseBlockQps(int count);
+    void increaseBlockQps(long rt, int count);
 
     /**
      * Add the biz exception count.
      *
+     * @param rt Current timestamp, calculated based on this
      * @param count count to add
      */
-    void increaseExceptionQps(int count);
+    void increaseExceptionQps(long rt, int count);
```

Comment:
This could bring breaking changes. Please describe why this is really needed.

Reply:
The request mentioned in #1039 is very close to the next second or high in the last second. I think rt should be monitored for each request when the status is changed.

Reply:
> The request mentioned in #1040 is very close to the next second or high in the last second. I think rt should be monitored for each request when the status is changed.

In your implementation, the RT is not actually used?

Reply:
I will resubmit